src/preferences/PreferencesManager.js

```diff
@@ -30,198 +31,16 @@
-
-    /**
-     * This method returns a standardized ClientID for a given requireJS module object
-     * @param {!{id: string, uri: string}} module - A requireJS module object
-     * @return {string} The ClientID
-     */
-    function getClientID(module) {
-        var paths = exports._getExtensionPaths();
-        var pathUrl, clientID;
-
-        paths.some(function (path) {
-            if (module.uri.toLocaleLowerCase().indexOf(path.toLocaleLowerCase()) === 0) {
-                pathUrl = path;
-                return true;
-            }
-        });
-
-        if (pathUrl) {
-            clientID = CLIENT_ID_PREFIX + module.uri.replace(pathUrl, "");
-        } else {
-            clientID = CLIENT_ID_PREFIX + module.id;
-        }
-        return clientID;
-    }
-
-    /**
-     * Retreive the preferences data for the given clientID.
-     * @deprecated
-     *
-     * @param {string|{id: string, uri: string}} clientID - A unique identifier or a requireJS module object
-     * @param {string=} defaults - Default preferences stored as JSON
-     * @param {boolean=} _doNotCreate Do not create the storage if it does not already exist. Used for conversion.
-     * @return {PreferenceStorage}
-     */
-    function getPreferenceStorage(clientID, defaults, _doNotCreate) {
-        // No one should be calling this to access the old preference storage except for
-        // migrating the old preferences to the new model. So if this is called without
-        // having _doNotCreate set to true, then the caller is using the old preferences model.
-        if (!_doNotCreate) {
-            var clientString = typeof clientID === "object" ? clientID.uri : clientID;
-            DeprecationWarning.deprecationWarning("PreferencesManager.getPreferenceStorage() called with client id '" +
-                                                  clientString + "' has been deprecated. Use PreferencesManager.definePreference() instead.");
-        }
-        if (!clientID || (typeof clientID === "object" && (!clientID.id || !clientID.uri))) {
-            console.error("Invalid clientID");
-            return;
-        }
-        if (typeof clientID === "object") {
-            clientID = getClientID(clientID);
-        }
-
-        var prefs = prefStorage[clientID];
-
-        if (prefs === undefined) {
-            if (_doNotCreate) {
-                return;
-            }
-            // create a new empty preferences object
-            prefs = (defaults && JSON.stringify(defaults)) ? defaults : {};
-            prefStorage[clientID] = prefs;
-        } else if (defaults) {
-            // add new defaults
-            _.forEach(defaults, function (value, key) {
-                if (prefs[key] === undefined) {
-                    prefs[key] = value;
-                }
-            });
-        }
-
-        return new OldPreferenceStorage(clientID, prefs);
-    }
-
-    /**
-     * Save all preference clients.
-     */
-    function savePreferences() {
```

Comment:
this method is being referenced for example here: https://github.com/adobe/brackets/blob/master/src/document/DocumentCommandHandlers.js#L1378

but also on multiple places